apps/main/src/llamalend/PageLlamaMarkets/cells/RateCell/BorrowRateTooltip.tsx

```diff
@@ -1,83 +1,43 @@
+import { useFilteredRewards } from '@ui-kit/shared/ui/tooltips/utils'
 import { useSnapshots } from '../../hooks/useSnapshots'
-import { formatPercent, useFilteredRewards } from '../cell.format'
-
-const { Spacing } = SizesAndSpaces
-
-const messages = {
-  [LlamaMarketType.Lend]: t`The borrow rate is the cost related to your borrow and varies according to the lend market, borrow incentives and its utilization.`,
-  [LlamaMarketType.Mint]: t`The borrow rate is the cost related to your borrow and varies according to the mint market, borrow incentives and the crvUSD's peg.`,
-}
 
 const rateType = 'borrow' as const
 
-const BorrowRateTooltipContent = ({ market }: { market: LlamaMarket }) => {
+export const BorrowRateTooltip = ({ market, children }: { market: LlamaMarket; children: ReactElement }) => {
   const {
-    chain,
     rewards,
     type: marketType,
-    rates,
-    rates: { borrow: borrowRate, borrowTotalApy },
+    rates: { borrow: borrowRate },
     assets: {
       collateral: { rebasingYield, symbol: collateralSymbol },
     },
   } = market
   const { averageRate, period } = useSnapshots(market, rateType)
-  const poolRewards = useFilteredRewards(rewards, marketType, rateType)
-  const extraIncentives = useMarketExtraIncentives(rateType, chain, rates)
+  const poolRewards = useFilteredRewards(rewards, marketType === 'Mint' ? 'mint' : 'lend', rateType)
 
   return (
-    <Stack gap={Spacing.sm}>
-      <Typography color="textSecondary">{messages[marketType]}</Typography>
-
-      {!!rebasingYield && (
-        <Typography color="textSecondary">{t`The collateral of this market is yield bearing and offers extra yield`}</Typography>
-      )}
-
-      <Stack>
-        <TooltipItems secondary>
-          <TooltipItem title={t`Borrow fees`}>{formatPercent(borrowRate)}</TooltipItem>
-        </TooltipItems>
-
-        {(poolRewards.length > 0 || extraIncentives.length > 0) && (
-          <TooltipItems secondary>
-            <RewardsTooltipItems title={t`Borrowing incentives`} {...{ poolRewards, extraIncentives }} />
-          </TooltipItems>
-        )}
-
-        {!!rebasingYield && (
-          <TooltipItems secondary>
-            <TooltipItem title={t`Yield bearing tokens`}>{formatPercent(rebasingYield)}</TooltipItem>
-            <TooltipItem subitem title={collateralSymbol}>
-              {formatPercent(rebasingYield)}
-            </TooltipItem>
-          </TooltipItems>
-        )}
-
-        <TooltipItems>
-          <TooltipItem primary title={t`Total borrow rate`}>
-            {formatPercent(borrowTotalApy)}
-          </TooltipItem>
-          <TooltipItem subitem loading={averageRate == null} title={`${period} ${t`Average`}`}>
-            {formatPercent(averageRate)}
-          </TooltipItem>
-        </TooltipItems>
-      </Stack>
-    </Stack>
+    <Tooltip
+      clickable
+      title={t`Borrow Rate`}
+      body={
+        <MarketBorrowRateTooltip
+          marketType={marketType === 'Mint' ? 'mint' : 'lend'}
```

Comment:
This will be wrong if we ever add another type, can we use a mapping instead?
Why do we mix lower and upper case anyway?